history.rb

```diff
@@ -3,7 +3,7 @@
 # frozen_string_literal: true
```

Comment:
NB: there's a typo in the commit message here "Explicity" → "Explicitly" (and "no actually" → "not actually")